lib/resty/upstream/healthcheck.lua

```diff
@@ -720,4 +720,66 @@ function _M.status_page()
     return concat(bits)
 end
 
+local function gen_peers_status_table(dict, peers, name, is_backup)
```

Comment:
I think `gen_peers_status_info` can be replaced with this function. What do you think @pushrax ?